quickwit/quickwit-indexing/src/actors/indexer.rs

```diff
@@ -49,6 +52,9 @@ use crate::models::{
     NewPublishLock, PreparedDoc, PreparedDocBatch, PublishLock, ScratchDirectory,
 };
 
+/// Limits the number of concurrent active workbenches to `num_cpus`.
+static INDEXING_PERMITS: Lazy<Semaphore> = Lazy::new(|| Semaphore::new(num_cpus::get()));
```

Comment:
num_cpus is not great here. We want the same logic as what is done in the `Runtimes`.

Reply:
you can find that in RuntimesType